Gemfile

```diff
@@ -163,6 +163,7 @@ group :test do
     gem 'term-ansicolor', '~> 1.3.0', '< 1.4'
   gem 'capybara', '~> 2.15.0'
   gem 'delorean', '~> 2.1.0'
+  gem 'stripe-ruby-mock', '~> 2.5.0'
```

Comment:
we should add the platform spec from the `stripe` gem here as well

Reply:
(not that that in itself will fix `rake spec` as it tries to load everything, finds `require 'stripe_mock'` and falls over but individual spec files will be runnable)

Reply:
Going to drop 1.9